src/widget/button.rs

```diff
@@ -123,6 +123,7 @@ impl Widget for Button {
     }
 
     fn paint(&mut self, cx: &mut PaintCx, builder: &mut SceneBuilder) {
+        println!("paint button with text {}", self.label);
```

Comment:
We probably want to not have such debug printing turned on by default. A more principled solution is to log this with `info!`, which is available because we have `tracing` as a dep.

Reply:
Oh right! Using tracing makes much more sense but i never used it. I probably should look into it